scripts/fast-reboot

```diff
@@ -170,26 +170,21 @@ function request_pre_shutdown()
     }
 }
 
-function recover_issu_bank_file_instruction()
+function recover_issu_bank_file()
 {
-    debug "To recover (${ISSU_BANK_FILE}) file, do the following:"
-    debug "$ docker exec -it syncd sx_api_dbg_generate_dump.py"
-    debug "$ docker exec -it syncd cat /tmp/sdkdump | grep 'ISSU Bank'"
-    debug "Command above will print the VALUE of ISSU BANK - 0 or 1, use this VALUE in the next command"
-    debug "$ printf VALUE > /host/warmboot/issu_bank.txt"
+    debug "Recovering the (${ISSU_BANK_FILE}) file"
+    docker exec -it syncd sx_api_dbg_generate_dump.py
```

Comment:
Can you change -it to -i. I think this change is needed if warm-reboot command was called from an ssh session.

Reply:
Done